sklearn/covariance/_graph_lasso.py

```diff
@@ -591,6 +592,13 @@ class GraphicalLassoCV(GraphicalLasso):
     grid_scores_ : ndarray of shape (n_alphas, n_folds)
         Log-likelihood score on left-out data across folds.
 
+        .. deprecated:: 0.21
+            The `grid_scores_` attribute is deprecated in version 0.23 in favor
+            of `cv_results_` and will be removed in version 0.25
+
+    cv_results_ : ndarray of shape (n_alphas, n_folds)
```

Comment:
Should we return a dict as in parameter search?

Reply:
yeah the API closer to what the `gs.cv_results_` looks like would make it easier for people to work with it I guess.